### DISCUSSION TITLE
FELRoot on Roborock S7 - device does not show up on lsusb

### DISCUSSION THREAD

**Discussion by mor3dr3ad** [Hypfer/Valetudo#1904] [Q&A / Support] [answered]
Hey, 

hopefully someone can help me on this: I am trying to root my Roborock S7 to flash valetudo and am failing miserably. I have disassembled the device and have access to the mainboard. Here is what I tried: 

- Attached small daughterboard to mainboard (the one with the buttons)
- pushed the "on" button for 3 seconds while connecting TPA17 to GND (I tried both SH1 as well as other GND pins) via jumper cable and/or paperclip. Before doing this I have scratched off the coating quite thoroughly
- When I manage to keep the pins connected, I am rewarded with a different LED-sequence: instead of the blue LEDs pulsating, I get a quick pulse and then 6 white LEDs before it moves to "wheels suspended" mode sometime. 
- However, the device does not show up on lsusb or the file manager. I have tried to different linux machines (arch & kali), no Win machine yet. 
- I am however on firmware 02.16.12 - could this be the reason? 

Thanks for any hints.

**Comment by flo269** [comment, marked as answer]
When we rooted we had the same issue and were clueless, until: We tried 3 different usb cables and with the third one it worked like a charm. Give that a try! :-)

**Reply by mor3dr3ad** [reply]
Already tried 3 different cables, the 4th one actually worked.. now I have to find a way around the firmware issue (02.16.12)

Thankx
